lib/node_modules/@stdlib/math/base/special/boxcoxinv/benchmark/python/benchmark.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+#
+# @license Apache-2.0
+#
+# Copyright (c) 2018 The Stdlib Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Benchmark scipy.special.inv_boxcox."""
+
+from __future__ import print_function
+import timeit
+
+NAME = "inv_boxcox"
```

Comment:
```suggestion
NAME = "boxcoxinv"
```

The typical convention is that the `NAME` variable should reflect the package name. This is so we can match benchmark TAP output with the corresponding JavaScript benchmark output.

Reply:
That makes sense. This was the first one that I worked on that the `scipy` function differs in name.

Reply:
Yeah, without docs (our failing) or a big picture POV, it's impossible to know about some of these conventions. We need to do better in removing the mystery of various conventions and practices.